en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md

```diff
@@ -18,72 +18,80 @@ The following diagram shows the overall solution we are going to build. The SMSC
-         <parameter name="addressTon">UNKNOWN</parameter>
-         <parameter name="systemType">CPT</parameter>
-         <parameter name="retryCount">-1</parameter>
-         <parameter name="bindType">BIND_RX</parameter>
-         <parameter name="addressRange">null</parameter>
-         <parameter name="systemId">kasun</parameter>
-         <parameter name="password">kasun</parameter>
-         <parameter name="exponentialFactor">5</parameter>
-         <parameter name="maximumBackoffTime">10000</parameter>
-      </parameters>
-   </inboundEndpoint>
-   ```
+      ```xml
+      <?xml version="1.0" encoding="UTF-8"?>
+      <sequence xmlns="http://ws.apache.org/ns/synapse" name="request" onError="fault">
+         <log level="custom">
+            <property xmlns:ns="http://org.apache.synapse/xsd"
+                      name="MessageId"
+                      expression="get-property('SMPP_MessageId')"/>
+            <property xmlns:ns="http://org.apache.synapse/xsd"
+                      name="SourceAddress"
+                      expression="get-property('SMPP_SourceAddress')"/>
+            <property xmlns:ns="http://org.apache.synapse/xsd"
+                      name="DataCoding"
+                      expression="get-property('SMPP_DataCoding')"/>
+            <property xmlns:ns="http://org.apache.synapse/xsd"
+                      name="ScheduleDeliveryTime"
+                      expression="get-property('SMPP_ScheduleDeliveryTime')"/>
+            <property xmlns:ns="http://org.apache.synapse/xsd"
+                      name="SequenceNumber"
+                      expression="get-property('SMPP_SequenceNumber')"/>
+            <property xmlns:ns="http://org.apache.synapse/xsd"
+                      name="ServiceType"
+                      expression="get-property('SMPP_ServiceType')"/>
+         </log>
+         <log level="full"/>
+      </sequence>
+      ```
    
-> **Note**: To configure the `systemId` and `password` parameter value, please use the steps given under the topic `Configure the SMSC (Short Message Service Center) simulator` in the [Setting up the SMPP Connector ]({{base_path}}/reference/connectors/smpp-connector/smpp-connector-configuration/) documentation.
-> - **systemId** : username to access the SMSC
-> - **password** : password to access the SMSC 
+3. Click `+` button next to `Inbound Endpoints` and select `SMPP` to add a new **SMPP inbound endpoint**.  
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/smpp-create-new-inbound-endpoint.png" title="Creating smpp inbound endpoint" width="1000" alt="Creating smpp inbound endpoint" style="border:1px solid black"/>
+    <br/>
+
+4. Configure the SMPP inbound endpoint as shown below and click `Create`.
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/smpp-config-endpoint-1.png" title="Configure SMPP inbound endpoint 1" width="1000" alt="Configure SMPP inbound endpoint 1" style="border:1px solid black"/>
```

Comment:
```suggestion
    <img src="{{base_path}}/assets/img/integrate/connectors/smpp-config-endpoint-1.png" title="Configure SMPP inbound endpoint 1" width="1000" alt="Configure SMPP inbound endpoint 1"/>
```

Please remove additional styling from all the images.

Reply:
resolved by https://github.com/wso2/docs-mi/pull/1094/commits/0e1b408369ae7139a985eb7ebb70d92547f19c12